### PR TITLE
Set version for slf4j and bindings

### PR DESCRIPTION
**Pull request by ao508**
PR explicitly sets the version of slf4j and its bindings to help prevent errors caused. by mismatched versions brought in by related dependencies. 

Signed-off-by: Angelica Ochoa <15623749+ao508@users.noreply.github.com>

### CODE REVIEW

**Comment by ao508** [comment]
This isn't going to resolve the issue we're seeing right now that you reported but we should do this here as well

File: pom.xml [hunk 1]
```diff
@@ -85,10 +88,34 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>2.5</version>
```
Comment (kalletlak) [inline pom.xml:105]: may be `${log4j.version}` similar to `${slf4j.version}`

**Review by kalletlak — APPROVED** [review]
LGTM

**Comment by kalletlak** [comment]
Btw it looks like these libraries are omitted in cbio because of conflicts. I think these changes are not required

**Comment by ao508** [comment]
thanks I'll go ahead and close the pr then!

**Pull request closed without merging**